src/commands/open.ts

```diff
@@ -0,0 +1,101 @@
+import * as _ from "lodash";
+import * as clc from "cli-color";
+import * as open from "open";
+
+import * as FirebaseError from "../error";
+import * as api from "../api";
+import * as Command from "../command";
+import * as logger from "../logger";
+import { promptOnce } from "../prompt";
+import * as requirePermissions from "../requirePermissions";
+import * as requireInstance from "../requireInstance";
+import * as utils from "../utils";
+
+interface Link {
+  name: string;
+  arg: string;
+  consoleUrl?: string;
+  url?: string;
+}
+
+const LINKS: Link[] = [
+  { name: "Analytics", arg: "analytics", consoleUrl: "/analytics" },
+  { name: "Authentication: Providers", arg: "auth", consoleUrl: "/authentication/providers" },
+  { name: "Authentication: Users", arg: "auth:users", consoleUrl: "/authentication/users" },
+  { name: "Crash Reporting", arg: "crash", consoleUrl: "/monitoring" },
+  { name: "Database: Data", arg: "database", consoleUrl: "/database/data" },
+  { name: "Database: Rules", arg: "database:rules", consoleUrl: "/database/rules" },
+  { name: "Docs", arg: "docs", url: "https://firebase.google.com/docs" },
```

Comment:
I'd also take this out into `FIREBASE_DOCS = "https://firebase.google.com"`

Reply:
Again, since it's not being used outside of exactly this context, pulling this out into a module constant doesn't make sense.

Reply:
I guess my problem with this is that sometimes the object has `url` and sometimes it has `consoleUrl` which makes it less consistent and necessitates special logic for the ones that don't follow the general pattern. If we were able to make this more consistent and reduce the special cases in the if/else down blow.

Reply:
I think with `url` vs `consolePath`, it makes more sense. If we did want to make it so that they all used `utils.consoleUrl`, we'd have to wrap _that_ in a function because it doesn't have a project (which that function depends on)